pkg/scm/git/git.go

```diff
@@ -217,83 +219,95 @@ func makePathAbsolute(source string) string {
 // expect git clone spec syntax; it also provides details if the file://
 // proto was explicitly specified, if we should use OS copy vs. the git
 // binary, and if a frag/ref has a bad format
-func ParseFile(source string) (details *FileProtoDetails, mods *URLMods) {
+func ParseFile(source string) (details *FileProtoDetails, mods *URLMods, err error) {
+	// NOTE: Why don't we just trim the source here?
```

Comment:
not sure what you mean. we're explicitly needing to track the difference between something like:
file://bparees@github.com  (user specified a file with an unusual name)
and
bparees@github.com (user specified an ssh syntax w/ no explicit protocol)

what are you proposing w/ trim?